en: add ticdc doc (#331)
pingcap/docs-tidb-operator: en/configure-cluster-using-tidbcluster.md

          count: 5
          level: information
```

## Configure TiCDC start parameters

> TiCDC start parameters can be configured by `spec.ticdc.config` in TidbCluster Custom Resource.

Reviewer: ```suggestion
You can configure TiCDC start parameters through `spec.ticdc.config` in TidbCluster Custom Resource.
```
